send_sms.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import json
+import general
+from config import URL, APIKEY, SECRETKEY, PHONENO
+
+# sending post request
+def send_post_request(reqUrl, apiKey, secretKey, useType, phoneNo, senderId, textMessage):
+  req_params = {
+  'apikey':apiKey,
+  'secret':secretKey,
+  'usetype':useType,
+  'phone': phoneNo,
+  'message':textMessage,
+  'senderid':senderId
+  }
+  return requests.post(reqUrl, req_params)
+
+
+def send_sms(message):
+    try:
+        # getting response and save as response object
+        response = send_post_request(URL, APIKEY, SECRETKEY, 'stage', PHONENO, 'WAYSMS', message )
+        # print response if you want
+        print (response.text)
+    except Exception as e:
+        print(e)
```

Comment:
it would be nice to print a better message along with the exception.
How about 
```suggestion
        print("Failed to send SMS: %s" % e)
```